scripts/notify-github-status.sh

```
@@ -0,0 +1,151 @@
+for pull_number in "${!pull_numbers[@]}"; do
+  target_numbers["${pull_number}"]=1
+  closing_issues="$(gh api graphql \
+    -f query='query($owner: String!, $repo: String!, $number: Int!) {
+      repository(owner: $owner, name: $repo) {
+        pullRequest(number: $number) {
+          closingIssuesReferences(first: 100) {
+            nodes { number }
+          }
+        }
+      }
+    }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)